code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -187,15 +196,14 @@ Made by Xhuis
 /datum/game_mode/proc/auto_declare_completion_shadowling()
 	var/text = ""
 	if(shadows.len)
-		text += "<br><font size=2><b>The shadowlings were:</b></font>"
+		text += "<br><font size=3><b>The shadowlings were:</b></font>"
```

Comment:
use spans so you can change them easier later

<span class="shadowlingcompletion"> blah </span>